src/applications/vaos/tests/list/va-appointments.unit.spec.jsx

```diff
@@ -0,0 +1,208 @@
+import React from 'react';
+import { expect } from 'chai';
+import moment from 'moment';
+import { merge } from 'lodash/fp';
+import { renderInReduxProvider } from 'platform/testing/unit/react-testing-library-helpers';
+import reducers from '../../reducers';
+import singleVAAppointment from '../mocks/single-va-appointment.json';
+import singleVAFacility from '../mocks/single-va-facility.json';
+import { mockAppointmentInfo, mockFacilitesFetch } from '../mocks/helpers';
+
+import FutureAppointmentsList from '../../components/FutureAppointmentsList';
+
+const initialState = {
+  featureToggles: {
+    vaOnlineSchedulingCancel: true,
+  },
+};
+
+describe('VAOS appointment list future booked', () => {
+  describe('va appointments', () => {
+    it('should show information without facility details', async () => {
+      const appointment = merge(singleVAAppointment, {
```

Comment:
I tried to set up the test days so that we had a basic structure we didn't need to repeat, but all the information actually be used in the test needs to be specified in the actual test code. That creates some repetition, but I think that's maybe worth it?

Reply:
Might be missing something but could we use the spread operator to clone singleVAAppointment and overwrite the fields we want to instead of using lodash merge?

Reply:
I made some changes here. I think having the mock data inside functions is safer overall. And that leaves it up to the person writing the tests how they want to set their data, without having to worry about shallow/deep clones and how merges work. 

I converted the ones in here to a combination of spread and direct setting, but I think the lodash merge function is fine if someone feels like that is more readable.